tests/Contents.m

```diff
@@ -0,0 +1,40 @@
+% TESTS Unit tests for Rigbox.
+% Version xxx 02-Oct-2019
```

Comment:
Why have this 'Version' comment here?

Reply:
Take a look at MATLAB's package and toolbox help documentation